doc/reference/reference_lua/config/utils_schema.rst

```diff
@@ -0,0 +1,1163 @@
+
+        *   -   :ref:`schema_object:merge() <config-schema_object-merge>`
+            -   Merge two configurations
+
+        *   -   :ref:`schema_object:pairs() <config-schema_object-pairs>`
+            -   Traverse a configuration
+
+        *   -   :ref:`schema_object:set() <config-schema_object-set>`
+            -   Set a configuration value
+
+        *   -   :ref:`schema_object:validate() <config-schema_object-validate>`
+            -   Validate a configuration against a schema
+
+        *   -   :ref:`schema_object.methods <config-schema_object-methods>`
+            -   User-defined methods
+
+        *   -   :ref:`schema_object.name <config-schema_object-name>`
+            -   Schema name
+
+        *   -   :ref:`schema_object.schema <config-schema_object-schema>`
+            -   Schema nodes hierarchy
+
+        *   -   **schema_node_annotation**
+            -
+
+        *   -   :ref:`allowed_values <config-schema_node_annotation-allowed_values>`
+            -   Allowed node values
+
+        *   -   :ref:`apply_default_if <config-schema_node_annotation-apply_default_if>`
+            -   Condition to apply defaults
+
+        *   -   :ref:`default <config-schema_node_annotation-default>`
+            -   Default node value
+
+        *   -   :ref:`type <config-schema_node_annotation-type>`
+            -   Value type
+
+        *   -   :ref:`validate <config-schema_node_annotation-validate>`
+            -   Validation function
+
+        *   -   **schema_node_object**
+            -
+
+        *   -   :ref:`schema_node_object.allowed_values <config-schema_node_object-allowed_values>`
+            -   Allowed node values
+
+        *   -   :ref:`schema_node_object.apply_default_if <config-schema_node_object-apply_default_if>`
+            -   Condition to apply defaults
+
+        *   -   :ref:`schema_node_object.computed <config-schema_node_object-computed>`
+            -   Computed annotations
+
+        *   -   :ref:`schema_node_object.default <config-schema_node_object-default>`
+            -   Default value
+
+        *   -   :ref:`schema_node_object.fields <config-schema_node_object-fields>`
+            -   Record node fields
+
+        *   -   :ref:`schema_node_object.items <config-schema_node_object-items>`
+            -   Array node items
+
+        *   -   :ref:`schema_node_object.type <config-schema_node_object-type>`
+            -   Scalar node type
+
+        *   -   :ref:`schema_node_object.validate <config-schema_node_object-validate>`
+            -   Validation function
+
+
+..  _config-utils-schema_functions:
+
+Functions
+~~~~~~~~~
+
+..  module:: config.utils.schema
+
+..  _config-utils-schema-array:
+
+..  function:: schema.array(opts)
+
+    Create an array node of a configuration schema.
+
+    :param table opts: a table in the following format:
+
+                       ..  code-block:: text
+
+                           { items = <schema_node_object>, <..annotations..> }
```

Comment:
Please, don't write 'concise' code by merging different things into a single line.

We have items and other annotations here. Let's format this code as usual for tables with several fields:

```lua
{
    items = <schema node>,
    <..annotations..>
}
```

Reply:
Fixed